unifdef/PKGBUILD

```
@@ -0,0 +1,23 @@
+# Maintainer: Joel Holdsworth <jholdsworth@nvidia.com>
+
+pkgname=unifdef
+pkgver=2.12
+pkgrel=1
+pkgdesc="Selectively processes conditional C preprocessor #if and #ifdef directives"
+arch=('any')
+url="https://dotat.at/prog/unifdef/"
+license=("BSD")
+makedepends=("gcc")
+source=("https://dotat.at/prog/${pkgname}/${pkgname}-${pkgver}.tar.xz")
+sha256sums=("43ce0f02ecdcdc723b2475575563ddb192e988c886d368260bc0a63aee3ac400")
+
+build() {
+  cd "${srcdir}"/${pkgname}-${pkgver}
+  make CC=${CHOST}-gcc all
```

`CC` is already set from /etc/makepkg-mingw.conf.

```suggestion
  make
```